packages/elements/src/tree-select/index.ts

```diff
@@ -45,11 +45,11 @@ const POPUP_POSITION = ['bottom-start', 'top-start'];
  * @attr {string} placeholder - Set placeholder text
  * @prop {string} [placeholder=""] - Set placeholder text
  * @prop {TreeSelectData[]} [data=[]] - Data object to be used for creating tree
- * @fires confirm - Fired when selection is confirmed
- * @fires cancel - Fired when selection is cancelled
- * @fires query-changed - Fired when query in input box changed
- * @fires value-changed - Fired when value of control changed
- * @fires opened-changed - Fires when opened state changes
+ * @fires confirm - Fired when user closed a popup with confirmation of new selection.
```

Comment:
"the" user close"s"~~d~~ a popup  by ~~with~~ confirming~~ation~~ ~~of~~ the new selection.